Semantics.md

```diff
@@ -42,7 +42,7 @@ environment such as a browser, a trap results in throwing a JavaScript exception
 If developer tools are active, attaching a debugger before the
 termination would be sensible.
 
-## Stack Overflow
+## Stack overflow
```

Comment:
Capitalization was inconsistent in the file, but seemed to be "Upper lower lower." Probably should be a different change though.